packages/ens-referrals/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@ensnode/ens-referrals",
+  "version": "0.36.0",
+  "type": "module",
+  "description": "A utility library for interacting with ENS Referrals data",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/namehash/ensnode.git",
+    "directory": "packages/ens-referrals"
+  },
+  "homepage": "https://github.com/namehash/ensnode/tree/main/packages/ens-referrals",
+  "keywords": [
+    "ENS",
+    "ENSReferrals"
```

```suggestion
    "ENS Referrals"
```

spaces ok here?

---

I believe we have go with no spaces. Perhaps just:
```suggestion
    "Referrals"
```